### PR TITLE
Documentation 2.0

### PR DESCRIPTION
**Pull request by Frank-Wiebeler**
Update Documentation to fit the new agrirouter implementation

### CODE REVIEW

File: modules/ROOT/assets/images/graphs/image15.SVG
Comment (oliverrahner) [inline modules/ROOT/assets/images/graphs/image15.SVG:1]: All file names should be lowercase

File: modules/ROOT/assets/images/graphs/image18.SVG
Comment (oliverrahner) [inline modules/ROOT/assets/images/graphs/image18.SVG:1]: One arrow remaining in old color
"No" branch should be green, "Yes" branch should be red

File: modules/ROOT/pages/commands/endpoint.adoc [hunk 1]
```diff
@@ -246,11 +246,10 @@ image::ig2/image45.png[,438,174]
 
 The teamset describes a list of devices attached to the communication unit. The most common devices are agricultural machines connected to the CU as ISO11783-10 TaskControllers. For more information on ISO11783-10, please refer to https://aef-online.org
 
-include::partial$device-description-has-to-be-sent-every-time.adoc[]
+## include::partial$device-description-has-to-be-sent-every-time.adoc[]
```
Comment (oliverrahner) [inline modules/ROOT/pages/commands/endpoint.adoc:249]: Add back

File: modules/ROOT/pages/commands/endpoint.adoc [hunk 2]
```diff
@@ -246,11 +246,10 @@ image::ig2/image45.png[,438,174]
...
 
 [NOTE]
-====
-For the certification it is required to send the device description message multiple times. Therefore, it must either be sent on manual request (request must be performable by the test processing person) or on reconnection/restart of the application.
+#For the certification it is required to send the device description message multiple times. Therefore, it must either be sent on manual request (request must be performable by the test processing person) or on reconnection/restart of the application.
```
Comment (oliverrahner) [inline modules/ROOT/pages/commands/endpoint.adoc:252]: Add back

File: modules/ROOT/pages/glossary.adoc [hunk 3]
```diff
@@ -80,16 +77,12 @@ DDIs are used, for example, in device descriptions to describe properties, the s
...
 
-agrirouter users can filter telemetry data for DDI Categories that abstract all DDIs into 10 categories. A list of DDIs and their Category assignment can be found here:
-
-A filtering for specific DDIs could not be provided, as this would have been too complex for the end user.
-
 
 |Live Telemetry Messages 
 |A live telemetry message contains data points with data from the machines in one teamset. Each data point at least contains the time when it was logged. Additionally, each data point can have a geo-position. Each data point may contain many log entries. Each log entry contains the value of a specific parameter (DDI) of a specific component or function (device element) of one of the machines in the teamset.
 
-The user-defined agrirouter routings define which parameters of which machine may go to which recipient. In addition, recipients can subscribe for certain parameters they want to get (sender-independent). Recipients will receive those data, when they are sent to the public address.
```
Comment (oliverrahner) [inline modules/ROOT/pages/glossary.adoc:91]: Re-add that first sentence, but put "... parameters from which (V)CU may go..."

File: modules/ROOT/pages/glossary.adoc [hunk 4]
```diff
@@ -80,16 +77,12 @@ DDIs are used, for example, in device descriptions to describe properties, the s
...
-
+Recipients can subscribe for certain parameters they want to get (sender-independent). Recipients will receive those data, when they are sent to the public address.
+## todo
```
Comment (oliverrahner) [inline modules/ROOT/pages/glossary.adoc:85]: Remove

File: modules/ROOT/pages/ids-and-definitions.adoc [hunk 5]
```diff
@@ -215,6 +215,10 @@ The gatewayId is an enumeration to determine the used protocol for all communica
 
 3=HTTP
 
+Switching GatewayID MQTT<-> HTTP works, but it may only be used for
```
Comment (oliverrahner) [inline modules/ROOT/pages/ids-and-definitions.adoc:218]: missing space (" <->")

File: modules/ROOT/pages/integration/authorization.adoc [hunk 6]
```diff
@@ -159,7 +159,8 @@ image::ig2/image14.png[Application authorization screen,333,303]
 
 agrirouter sends an HTTP 301 redirect to the browser, encoding the authorization result in a GET queue attached to the Redirect URL entered in the developers' application settings.
 
-The browser reacts in requesting this redirect URL which performs a GET request at the endpoint of the address.
+The browser reacts in requesting this redirect URL which performs a GET request at the endpoint of the address. The first redirect URL set comes into
+play when an authorize request does not provide a redirect URL (or provides an invalid one).
```
Comment (oliverrahner) [inline modules/ROOT/pages/integration/authorization.adoc:163]: "Please note that this behavior is DEPRECATED and we will require a VALID redirect URL to be provided to an authorize request in the future."
--> Please also note this in the "Parameters" section

File: modules/ROOT/pages/integration/onboarding.adoc [hunk 7]
```diff
@@ -52,7 +52,8 @@ From the end user’s perspective, it is mostly dependent on the telemetry platf
 
 === Using the agrirouter UI
 
-The registration code for a new CU can be created by the end user clicking "Onboard Telemetry Unit" in agrirouter's control center. He has to select the desired CU  and gets a 10-digit code consisting of letters and numbers. A CU needs an interface to input this registration code.
+The registration code for a new CU can be created by the end user clicking "Onboard Telemetry Unit" in agrirouter's control center. He gets a 10-digit code consisting of letters and numbers. A CU needs an interface to input this registration code. Registration Codes are
```
Comment (oliverrahner) [inline modules/ROOT/pages/integration/onboarding.adoc:55]: Button should be called "+ Connection"

File: modules/ROOT/pages/limitations.adoc [hunk 8]
```diff
@@ -10,7 +10,6 @@ The number of endpoints in agrirouter is limited as to:
 | Applications (CUs / telemetry platforms / farming software) | 300
 | Machines | 300
```
Comment (oliverrahner) [inline modules/ROOT/pages/limitations.adoc:11]: also needs to be removed

File: modules/ROOT/pages/limitations.adoc [hunk 9]
```diff
@@ -10,7 +10,6 @@ The number of endpoints in agrirouter is limited as to:
...
 | Virtual CUs | 100
-| Paired Accounts (incoming or outgoing)| 100 
 | Groups | 20
```
Comment (oliverrahner) [inline modules/ROOT/pages/limitations.adoc:13]: also needs to be removed

File: modules/ROOT/pages/messaging-workflow.adoc [hunk 10]
```diff
@@ -169,7 +169,7 @@ the protocol layer of REST or MQTT.
 With except to the EFDI telemetry messages (DeviceDescription and timelogs), agrirouter doesn’t analyse the messages inside the agrirouter command. It just checks the message type and the addressing to determine the recipients based on the routings and subscriptions.
```
Comment (oliverrahner) [inline modules/ROOT/pages/messaging-workflow.adoc:169]: With the exception of...

**Review by oliverrahner — CHANGES_REQUESTED** [review]
All file

**Comment by Frank-Wiebeler** [comment]
Closed as the merge was done via a different branch to ensure the pipelines run

**Comment by Frank-Wiebeler** [comment]
Closed as it's not needed anymore

**Pull request closed without merging**